nmosauth/auth_server/security_service.py

```diff
@@ -41,12 +41,23 @@
 
 class SecurityService:
     def __init__(self, logger=None):
-        self.config = _config
+        self.config = {"priority": 100, "https_mode": "disabled", "enable_mdns": True}
```

Comment:
It doesn't look like you're actually using 'enable_mdns'. It would probably be useful, so if you can use it in the same way as the Reg/Query API that would be good.